docs/release-notes/rl-0.4.adoc

```diff
@@ -20,6 +20,8 @@ https://github.com/n3oney[n3oney]:
 
 * Refactored the resolveDag function - you can just provide a string now, which will default to dag.entryAnywhere
 
+* Fixed formatting issues and gave null-ls priority over other formatters
+
```

Comment:
I'd say this is ready for a merge after proper testing, but please do elaborate on "formatting issues" here.

Reply:
i put it there for #77

Reply:
As in within the documentation

Reply:
what